gflownet/proxy/crystals/lattice_parameters.py

```diff
@@ -0,0 +1,32 @@
+import pickle
+from pathlib import Path
+from typing import List
+
+import numpy as np
+from torch import Tensor
+
+from gflownet.proxy.base import Proxy
+
+PICKLE_PATH = Path(__file__).parents[3] / "data" / "crystals" / "lattice_proxy.pickle"
+
+
+class LatticeParameters(Proxy):
+    def __init__(self, min_value: float = -100, **kwargs):
+        super().__init__(**kwargs)
+
+        self.min_value = min_value
+        self.kde = None
+
+    def __call__(self, states: List) -> Tensor:
+        scores = self.kde.score_samples(states)
+        scores = np.clip(scores, self.min_value, np.inf)
+        scores = self.min_value - scores
```

Comment:
Does this work fine? The convention for the proxies is that they are in a range [-MIN, 0).

Reply:
Well, it should. Original scores should be in `(-inf, 0)` (in practice, for the discrete grid states, that will be `~(-700, 0)`). Clipping will reduce that to `[-100, 0)`, so `self.min_value - scores = -100 - scores` will still be in `(-100, 0]` range, just in the reversed order. But let me know if you see a problem with this.

Reply:
Ok got you thanks!